README.md

```diff
@@ -57,6 +57,17 @@ To deploy the contract on Anvil, run:
 forge create --rpc-url http://127.0.0.1:8545 --mnemonic "test test test test test test test test test test test junk" src/Spark.sol:Spark --constructor-args 0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266
 ```
 
+## Node.js API
+
+```js
+import * as Spark from '@filecoin-station/spark-impact-evaluator'
+
+console.log({
+  abi: Spark.ABI,
+  address: Spark.ADDRESS
+})
```

Comment:
Nitpick - feel free to ignore.

I find the name `Spark` too vague; it can mean anything. I would choose a name to describe this variable as a configuration of a smart contract.  Something like `IeContract`, `SparkIeContract` or perhaps `SparkImpactEvaluator` (this last one also matches the package name).

```suggestion
import * as SparkImpactEvaluator from '@filecoin-station/spark-impact-evaluator'

console.log({
  abi: SparkImpactEvaluator,
  address: SparkImpactEvaluator
})
```

Reply:
```suggestion
import * as SparkImpactEvaluator from '@filecoin-station/spark-impact-evaluator'

console.log({
  abi: SparkImpactEvaluator.ABI,
  address: SparkImpactEvaluator.ADDRESS
})
```